codegenerator/cli/templates/static/codegen/src/EventProcessing.res

```diff
@@ -283,6 +283,14 @@ let addEventToRawEvents = (eventItem: Internal.eventItem, ~inMemoryStore: InMemo
     params
     ->S.reverseConvertOrThrow(paramsRawEventSchema)
     ->(Utils.magic: unknown => Js.Json.t)
+  let params = if params === %raw(`null`) {
+    // Should probably make the params field nullable
+    // But this is currently needed to make events
+    // with empty params work
+    %raw(`"null"`)
+  } else {
```

Comment:
I know it's dirty, but this is what we did for fuel. Actually, with the change I moved some code from `coerceToJsonPgType` which made things a little bit simpler. 
Ideally, all the raw_events table schema should be configured in a config and managed automatically by our db framework (which we discussed with Jono but haven't implemented)